gulpfile.js

```diff
@@ -11,7 +11,7 @@ var typescript = require('typescript');
 var mocha = require('gulp-mocha');
 
 gulp.task('check-format', function() {
-  return gulp.src(['*.js', 'lib/**/*.ts', 'test/**/*.ts'])
+  return gulp.src(['*.js', 'lib/**/*.ts', 'test/**/*.ts', '!test/fixtures/**/*.ts'])
```

Comment:
I see. sgtm

On Mon, Jun 20, 2016 at 8:53 AM Jason Choi <notifications@github.com> wrote:

> In gulpfile.js
> <https://github.com/angular/ts-api-guardian/pull/6#discussion_r67714665>:
>
> > @@ -11,7 +11,7 @@ var typescript = require('typescript');
> >  var mocha = require('gulp-mocha');
> >
> >  gulp.task('check-format', function() {
> > -  return gulp.src(['*.js', 'lib/**/*.ts', 'test/**/*.ts'])
> > +  return gulp.src(['*.js', 'lib/**/*.ts', 'test/**/*.ts', '!test/fixtures/**/*.ts'])
>
> The serializer depends on the whitespace determinism of generated d.ts
> files, so disabling formatting reflects this. For example, the formatter
> makes short declarations like interface A { b: number; } one line, which
> never happens in a generated d.ts file.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/angular/ts-api-guardian/pull/6/files/6dc6beb0b16b8a8af79383784bbe54779104e8ce#r67714665>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AANM6JjFn4DpxO0WUiFRw6sWihSqBgqkks5qNrdmgaJpZM4I40lA>
> .
>